en/components/navbar.md

```diff
@@ -43,26 +43,18 @@ Then in the template of our component we can add the following code to show a ba
 </igx-navbar>
 ```
 
-####With back button
+#### With menu button
 Good, we know which application we have opened. Now, let's see what capabilities it offers us by exploring its menu. To achieve this we will show the action button and make it use a menu icon as follows:
 
 ```html
 <!--navbar.component.html-->
 
-<igx-navbar title="Sample App"
-    actionButtonIcon="menu"
-    [isActionButtonVisible]="true">
+<igx-navbar title="Sample App" actionButtonIcon="menu" [isActionButtonVisible]="true">
 </igx-navbar>
```

Comment:
We should keep the simple first sample here so they can see the navbar with the menu button and title.